admin/app/webpack/javascripts/components/node-form.es6

```diff
@@ -56,9 +56,29 @@ export default Vue.component('node-form', {
   methods: {
     setSlug: function() {
       if (this.node['new_record?']) {
-        this.node.slug = this.node.title.toLowerCase().replace(/[\s\_]/g, '-').replace(/[^\w\-]/g, '');
+        this.node.slug = this.node.title.toLowerCase();
```

Comment:
👍 

Also, probably slightly cleaner style to chain it all together, eg:

~~~javascript
this.node.slug = this.node.title.toLowerCase()
                  .replace(/[àáâãäå]/g,"a")
                  .replace(/[éèêëęėē]/g,"e")
                  // etc
                  .replace(/[\s\_]/g, '-').replace(/[^\w\-]/g, '');
~~~